demo/config.yaml

```diff
@@ -171,3 +171,9 @@ lcr-modules:
         # include here any additional flags to modify default parameters
         options:
             sage_run: ""
+
+
+    gatk_rnaseq:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
```

Comment:
This needs new line added at the end

Reply:
Done

Reply:
It still shows no new line here, maybe I am looking at outdated file?

Reply:
Sorry, I must have missed it. It's fixed